src/query/ast/src/ast/format/ast_format.rs

```diff
@@ -22,8 +22,8 @@ use itertools::Itertools;
 
 use crate::ast::*;
 
-pub fn format_statement(stmt: Statement) -> Result<String> {
-    let mut visitor = AstFormatVisitor::new();
+pub fn format_statement(stmt: Statement, ignore_literal: bool) -> Result<String> {
+    let mut visitor = AstFormatVisitor::new(ignore_literal);
```

Comment:
`format_statement` is used to generate the tree structure, here just used to rewrite the literal. We can write a separate `Visitor` to rewrite literal, and then converted to string.

Reply:
Done